easybuild/easyconfigs/p/pubtcrs/pubtcrs-f30b96a-intel-2019.01.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'MakeCp'
+
+name = 'pubtcrs'
+version = 'f30b96a'
```

Comment:
@sassy-crick We should replace this with the datestamp of the commit as version, and also rename the source tarball on download...